pkg/META.in

```diff
@@ -10,7 +10,6 @@ archive(syntax, preprocessor, utf8) = "-ignore"
 preprocessor = "ocamlreason_preprocessor_where_does_this_end_up"
 
 # Toplevel
-archive(byte, toploop) = "@compiler-libs/ocamlcommon.cma @str/str.cma"
```

Comment:
I'm guessing you broke something somewhere :D

We'll just wait and find out what it is.

Does `rtop` still work for you after this?